tests/src/tools/ReadyToRun.SuperIlc/CompileDirectoryCommand.cs

```diff
@@ -87,6 +96,17 @@ public static int CompileDirectory(DirectoryInfo toolDirectory, DirectoryInfo in
                 }
             }
 
+            Console.WriteLine($"Compiled {successfulCompileCount} assemblies.");
+
+            if (_failedCompilationAssemblies.Count > 0)
+            {
+                Console.WriteLine($"Could not compile the following assemblies:");
```

Comment:
Nit - could you please also display the number of failing assemblies so that we don't need to count the lines in case of a larger number of failures?

Reply:
Absolutely